### PR TITLE
Fixing Theming for webviews. 

### PR DESCRIPTION
**Pull request by aasimkhan30**
Connection Dialog in different themes.
image

### CODE REVIEW

File: src/reactviews/common/utils.ts [hunk 1]
```diff
@@ -29,21 +25,21 @@ export function formatString(str: string, ...args: any[]): string {
 
 /**
  * Gets the theme type based on the theme passed in
- * @param theme the theme of the react webview
+ * @param themeKind the theme of the react webview
```
Comment (Benjin) [inline src/reactviews/common/utils.ts:28]: Can you update the comments on this (and related) utility methods that state what they're mapping from/to?  e.g. "maps from VS Code theme type to Fluent theme type", with a link to the docs page that says what the valid fluent types are.

File: src/reactviews/common/utils.ts [hunk 2]
```diff
@@ -29,21 +25,21 @@ export function formatString(str: string, ...args: any[]): string {
...
-        case webDarkTheme:
+export function getVscodeThemeType(themeKind: ColorThemeKind): string {
+    switch (themeKind) {
+        case ColorThemeKind.Dark:
             return "vs-dark";
-        case teamsHighContrastTheme:
+        case ColorThemeKind.HighContrast:
             return "hc-black";
         default:
             return "light";
```
Comment (Benjin) [inline src/reactviews/common/utils.ts:37]: Do we care about the high-contrast light theme in VS Code?  Or maybe Fluent doesn't have an appropriate mapping for it?

**Review by Benjin — APPROVED** [review]

File: src/reactviews/common/utils.ts (hunk 1 above)
Comment (aasimkhan30) [reply src/reactviews/common/utils.ts:28]: This doesn't do any mapping now. It is just exposing the vscode theme kind.

File: src/reactviews/common/utils.ts (hunk 2 above)
Comment (aasimkhan30) [reply src/reactviews/common/utils.ts:37]: I am using this in the table designer monaco editor which doesn't implement the hc-light yet. That's why I am just handling hc-light as light.

**Review by kburtram — APPROVED** [review]
looks great!

**Pull request merged**